assets/css/main.css

```diff
@@ -19,11 +19,13 @@ body {
 Header/Navbar
  */
 
-header {
+header,
+.general-page-bg {
 	background-image: url("/theme-assets/homepage/endwalker@2x.png");
```

Comment:
not right now, but I'd like to see optional frontmatter overload for a header image at some point.